test/screenshot/config.js

```diff
@@ -36,6 +36,8 @@ export const buttonConfigs : Array<ButtonConfig> = [];
 const DEFAULT_OAUTH_PROPS = {
     scopes:          [ 'email' ],
     responseType:   'code',
+    inputLabel: 'connect',
```

Comment:
This should be `style: { label: 'connect' }` right?

Reply:
done